wdl-gauntlet/Cargo.toml

```diff
@@ -24,7 +22,6 @@ serde_with.workspace = true
 temp-dir = "0.1.13"
 tokio.workspace = true
 toml.workspace = true
-urlencoding = "2.1.3"
-wdl-ast = { path = "../wdl-ast", version = "0.1.0" }
-wdl-core = { path = "../wdl-core", version = "0.1.0" }
-wdl-grammar = { path = "../wdl-grammar", version = "0.2.0" }
+wdl-ast = { path = "../wdl-ast" }
```

Comment:
@peterhuene can you comment on this change? Andrew asked me over Slack if this was necessary and I have no idea lol. It would depend on how `cargo` makes decisions about building and that's a mystery to me.

Reply:
My understanding of the way `cargo` works is that if you're building normally via `cargo build`, a `path` dependency will _always_ build from the local version of the crate, regardless of a specified `version`.

However, if you intend to publish a crate with a `path` dependency, `cargo` needs to know the published crate version to use a substitute for the local path as it will _not_ build anything outside of the crate building published.

So if we intend to publish `wdl-gauntlet` to crates.io, you will need the versions specified; otherwise, you won't need them.

Reply:
> So if we intend to publish wdl-gauntlet to crates.io, you will need the versions specified; otherwise, you won't need them.

It appears Clay beat us to the punch: https://crates.io/crates/wdl-gauntlet
Although me and Andrew were saying earlier that we don't know who else would want to use this crate 😅 
Should I put those paths back?

Reply:
I think we might as well.